bucket/pyenv.json

```diff
@@ -12,10 +12,11 @@
         }
     },
     "pre_install": "if (!(Test-Path \"$persist_dir\\pyenv-win\\version\")) { New-Item \"$dir\\pyenv-win\\version\" | Out-Null }",
-    "bin": "pyenv-win\\bin\\pyenv.bat",
-    "env_add_path": "pyenv-win\\shims",
+    "env_add_path": ["pyenv-win\\bin", "pyenv-win\\shims"],
```

Comment:
```suggestion
    "env_add_path": [
        "pyenv-win\\bin",
        "pyenv-win\\shims"
    ],
```

Reply:
Oh thanks. I'll remember to correctly format like this always.